File: my-folder/problems/running_sum_of_1d_array/solution.java
class Solution {
    int febo(int nums[], int i){
        int ans=0;
        for(int j=0; j<=i; j++){
            ans+= nums[j];
        }
        return ans;
    }
    public int[] runningSum(int[] nums) {
        int [] result= new int[nums.length];
        result[0]= nums[0];
        for(int i=1; i<nums.length; i++){
            result[i]= febo(nums,i);
        }
        
        return result;
    }
}

File: my-folder/problems/running_sum_of_1d_array/solution.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        total= sum(nums)
        n= len(nums)
        nums1= nums.copy()
        nums1[n-1]= total
        for i in range(n-2, -1, -1):
            total= total-nums[i+1]
            nums1[i]= total
            
        return nums1